misc/scripts/error_log.sh

```diff
@@ -31,7 +31,7 @@ function error_log() {
 	local code="${1}"
 	local scope="${2}"
 
-	if [[ ! -f "$LOGFILE" ]]; then
+	if [[ ! -f $LOGFILE ]]; then
```

Comment:
Quote this.

Reply:
Okay the unquoting is due to the `-s` flag. You sure you want to get rid of the `-s` flag though? It has other benefits. Check the shfmt docs.